### PR TITLE
reformatted the layout, updated progress

### PR DESCRIPTION
**Pull request by paradite**

### CODE REVIEW

**Comment by lowjoel** [comment]
Was the architecture overview in your presentation on Monday?

File: teams/socketio.md [hunk 1]
```diff
@@ -18,43 +18,33 @@ permalink: /teams/socketio/
...
- - @mikelimantara
-
- - @YijinL
-
 ## Workplan
 
 * [x] Project set up - Week 1, Mon
 * [x] Domain understood - Week 1, Thu
+* [x] Product demo using deployed chat app - Week 2, Mon
+* [x] High level architecture understanding - Week 2, Thu
```
Comment (damithc) [inline teams/socketio.md]: 'mark as done' an item only after it is done.

File: teams/socketio.md [hunk 2]
```diff
@@ -18,43 +18,33 @@ permalink: /teams/socketio/
...
+### @paradite (Zhu Liang)
 
-##### Contribution to project SI
-* ...
+#### Contribution to project SI
+* Set up own chat app using socket.io - Week 1
+* Draw architecture overview for socket.io - Week 2
 
-#### Technology expertise in tool Z
+#### Technology expertise in Angular 2.0
```
Comment (damithc) [inline teams/socketio.md]: Just `Angular`? May be shouldn't tie yourself to a particular version.

Comment (paradite) [inline teams/socketio.md]: okay

**Comment by paradite** [comment]
updated the pull request after today's session

File: teams/socketio.md [hunk 3]
```diff
@@ -18,43 +18,32 @@ permalink: /teams/socketio/
...
 
-#### @paradite (Zhu Liang)
-* [ ] ...
+### @paradite (Zhu Liang)
 
-##### Contribution to project SI
-* ...
+#### Contribution to project SI
+* Set up own chat app using socket.io - Week 1
+* Draw architecture overview for socket.io - Week 2
```
Comment (lowjoel) [inline teams/socketio.md:34]: This should be in your presentation next Monday.

**Pull request merged**